neo/Storage/Implementation/LevelDB/LevelDBClassMethods.py

```diff
@@ -0,0 +1,235 @@
+import plyvel
+import threading
+
+from contextlib import contextmanager
+
+from neo.Core.Blockchain import Blockchain
+from neo.Storage.Common.DBPrefix import DBPrefix
+from neo.Storage.Interface.DBInterface import DBProperties
+from neo.logging import log_manager
+
+
+logger = log_manager.getLogger()
+
+"""
+Description:
+    All the methods are used within the dynamically generated class from the
+    database factory.
+Usage:
+    For a new database implementation all methods defined in
+    AbstractDBImplementation have to be implemented.
```

Comment:
I think the same comment applies here as I did on the `AbstractDBImplementation`. I'm aware that we refer to this module as an example how to add a new backend. Therefore I'm fine with adding some more words that are backend generic/howto. E.g.
```
Description:
    A backend implementation for the LevelDB database.

   It overrides all methods from the `AbstractDBImplementation` class. The database factory (`DBFactory`) uses these methods to dynamically generate a conforming database instance for internal usage.
```

Reply:
Added your description :+1: